app/directives/forms/edit/edit-side-event.js

```diff
@@ -283,8 +293,8 @@ define(['app', 'lodash',
 
                         function findOpenRegsQuery(){
 
-                            const q = isNew()? { '$or' : [ { institution: 'CBD' }, { institution: 'cbd' } ], schedule : { $exists: true }, 'schedule.sideEvents.start'  : { $lte: { $date: moment.utc() } }, 'schedule.sideEvents.end'    : { $gt: { $date: moment.utc() } } } :
-                                                        { '$or': [ { institution: 'CBD' }, { institution: 'cbd' } ], schedule: { $exists: true }, EndDate: { $gt: { $date: moment.utc() } } }
+                            const q = { $or: [{ '$or' : [ { institution: 'CBD' }, { institution: 'cbd' } ], schedule : { $exists: true }, 'schedule.sideEvents.start'  : { $lte: { $date: moment.utc() } }, 'schedule.sideEvents.end'    : { $gt: { $date: moment.utc() } } },
```

Comment:
its wrong, if the institution is not cbd and has schedule for other institution it will appear